libcxx/include/__format/format_functions.h

```diff
@@ -63,14 +63,15 @@ using wformat_args = basic_format_args<wformat_context>;
 #  endif
 
 template <class _Context = format_context, class... _Args>
-_LIBCPP_NODISCARD_EXT _LIBCPP_HIDE_FROM_ABI __format_arg_store<_Context, _Args...> make_format_args(_Args&... __args) {
+_LIBCPP_NODISCARD_EXT _LIBCPP_HIDE_FROM_ABI __format_arg_store<_Context, _Args...>
+make_format_args(_Args&... __args _LIBCPP_LIFETIMEBOUND) {
```

Comment:
```suggestion
make_format_args(_LIBCPP_LIFETIMEBOUND _Args&... __args) {
```
Same below, just to keep it consistent across the code base.